accepted/2025/rid-specific-tool-packages.md

```diff
@@ -0,0 +1,91 @@
+# RID-Specific .NET Tool Packages
+
+**Owner** [Daniel Plaisted](https://github.com/dsplaisted)
+
+## Summary
+
+This proposal adds the ability for a .NET Tool to have separate packages for each supported OS and architecture.
+
+In .NET, the combination of OS and architecture is represented by a Runtime Identifier (RID).  Today, .NET Tools support native assets, but the native assets for all of the supported Runtime Identifiers need to be included in the same package.  For tools with large native dependencies, or if the entire tool is native (for example via NativeAOT), this multiplies the package download size by the number of supported RIDs.
+
+We would like to add support for RID-specific .NET Tool packages.  This would enable a tool to have a separate package for each supported RID, and when the tool is installed only the package for the correct RID would need to be downloaded.
+
+### Goals
+
+- Tools can create RID-specific packages
+- Installing a tool with RID-specific packages is transparent but only downloads the package for the current RID
```

Comment:
It will download both the top level (generic) package and the RID specific package (2) right? Not 1?

Reply:
That's correct.  I didn't find a way to word this to be precisely correct without being confusing.